src/messages.ts

```diff
@@ -0,0 +1,30 @@
+import * as fs from 'fs';
+import * as pt from 'path';
+import * as format from 'string-format';
```

Comment:
Failed to run 'npm run all' due to missing type definitions for 'string-format'.